data/features/m/m.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+m() {
+  if command -v man >/dev/null; then
+    if [[ $# -eq 1 ]]; then
+      man_output=$(man "$@" | grep -E '^ +-[a-zA-Z], --[a-zA-Z-]+')
+      echo "$man_output"
+    elif [[ $# -eq 2 ]]; then
```

Comment:
Redo this so we process any number of arguments. Just do a `for`, for each argument after the first one.

Reply:
so think like  `m ls -h -l -A` will work

Reply:
solved